qa/L0_python_api/test.sh

```diff
@@ -49,6 +49,15 @@ if [ $? -ne 0 ]; then
     RET=1
 fi
 
+
+FRONTEND_TEST_LOG="./python_tritonfrontend_api.log"
+python -m pytest --junitxml=test_kserve_frontend.xml test_kserve_frontend.py > $FRONTEND_TEST_LOG 2>&1
```

Comment:
Is the filename right? Looks like it is `test_kserve.py` now, please double check.

Reply:
You can run `bash -ex test.sh` locally to verify your test scripts are working instead of waiting for a pipeline.

Reply:
Thanks for pointing it out. Will fix right now.